osu.Game/Screens/Menu/ButtonArea.cs

```diff
@@ -20,7 +20,7 @@ public class ButtonArea : Container, IStateful<Visibility>
         private readonly ButtonAreaBackground buttonAreaBackground;
         private Visibility state;
 
-        public const float BUTTON_AREA_HEIGHT = 100;
+        public const int BUTTON_AREA_HEIGHT = 100;
```

Comment:
Why? Height is of type `float`, not `int`

Reply:
I'm using int for consts as well, so no judging here